zcash_primitives/Cargo.toml

```diff
@@ -19,8 +19,13 @@ pairing = { path = "../pairing" }
 rand = "0.7"
 rand_core = "0.5"
 rand_os = "0.2"
+ripemd160 = { version = "0.8", optional = true }
+secp256k1 = { version = "=0.15.0", optional = true }
```

Comment:
Dependency is pinned until https://github.com/rust-bitcoin/rust-secp256k1/issues/150 is resolved.

Reply:
Have they fixed this yet? (Issue isn't closed but they seem to have merged some things.)

Reply:
They merged a fix, and then immediately reverted it because they had more point releases to make, and the fix for this requires a minor version bump.